dom.bs

```diff
@@ -5154,6 +5170,10 @@ invoked, must run these steps:
 
  <li><p>Initialize <var>event</var>'s {{Event/type}} attribute to the empty string.
 
+ <li><p>Initialize <var>event</var>'s {{Event/timeStamp}} attribute to a {{DOMHighResTimeStamp}}
+ which is the high resolution time from the <a>time origin</a> to the occurrence of the call to
+ {{Document/createEvent()}}.
```

Comment:
This was what I was talking about with respect to duplication, but I guess that's also following the isTrusted precedent. Probably okay. Can this become "from the time origin to now"? It's a little weird for the algorithm to refer to itself this way.